content/en/docs/apidocs-mxsdk/apidocs/design-properties.md

```diff
@@ -90,40 +103,96 @@ Use this code to further simplify the above example:
 }
 ```
 
-As you can see from the structure above, design properties are defined as a *JSON* object where keys (`DivContainer` and `Button` for example) are widget types and values are *JSON* arrays containing sets of design properties applicable for that widget type. To learn more about this subject, see the [Widget Types](#widget-types) section below. Note that you can add multiple class names in a property.
+As you can see, design properties are defined as a *JSON* object where the keys (`DivContainer` and `Button` for example) are widget types and the values are *JSON* arrays containing sets of design properties applicable for that widget type. To learn more about this subject, see the [Widget Types](#widget-types) section below.
+
+Every design property in the array is also represented by a *JSON* object. The exact structure of each object is dependent on its type, as indicated by the `type` property. All types also share common fields such as `name` and `description`. Those names determine how a design property appears to a user in Mendix Studio Pro.
+
+#### 4.1.1 Common Fields
 
-Every design property from the array is also represented as a *JSON* object. As described earlier there are two types of design properties: **Toggle** and **Dropdown**. Both types share common fields such as `name`, `type`, and `description`. Those names determine how a design property appears to a user in Mendix Studio Pro. Below you can look at examples of design properties of both types.
+The examples above show that the fields `name` and `description` define the UI, the name of a form control in Studio Pro, and the description under it. They are arbitrary string values naming and describing a design property. 
```

Comment:
I understand that your trying to say the that the contents of these values can be any string, but this might be look like we're saying that these fields aren't mandatory